src/TwoSlotDisplay.js

```diff
@@ -34,11 +42,13 @@ class TwoSlotDisplay extends Component {
 
   render() {
     let next = getNextSlot(this.state.day, this.state.slot);
+    const weekend = this.state.day === 0 || this.state.day === 6;
     return (
       <div className="container">
         <h3>
           Choose the slot, area and floor that you want to chill at:
         </h3>
+        <h5>{ weekend &&  "PS: You shouldn't be attending classes on weekends! :P" }</h5>
```

Comment:
This will lead to an empty `h5` tag when `weekend` is `false`. Better to use an `if` to ensure that the tag doesn't show up at all when `isWeekend` is false.